ddspipe_yaml/test/unittest/yaml_reader/log_conf/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+# Copyright 2024 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+###############################
+# Yaml Reader Log Config Test #
```

Comment:
NIT: I would like it better if it was called `Log Configuration` everywhere instead of `Log Config`.

Reply:
Please rename also the file name and the folder.